superbench/benchmarks/model_benchmarks/pytorch_lstm.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the Pytorch LSTM model."""
+
+import time
+
+import torch
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, Precision
+from superbench.benchmarks.model_benchmarks.model_base import Optimizer
+from superbench.benchmarks.model_benchmarks.pytorch_base import PytorchBase
+from superbench.benchmarks.model_benchmarks.random_dataset import TorchRandomDataset
+
+
+class LSTMBenchmarkModel(torch.nn.Module):
```

Comment:
`LSTMModel`? it's not related to benchmark itself.

Reply:
torch.nn.LSTM is LSTM model. Since I added linear layer for LSTM, the current LSTM model is just for benchmarking.